spec/operators/share-spec.js

```diff
@@ -0,0 +1,50 @@
+/* globals describe, expect, it, hot, cold, expectObservable */
+
+var Rx = require('../../dist/cjs/Rx');
+var Observable = Rx.Observable;
+
+describe('Observable.prototype.share()', function (){
+  it('should share a single subscription', function (){
+    var subscriptionCount = 0;
+    var obs = new Observable(function(observer){
+      subscriptionCount++;
+    });
+    
+    var source = obs.share();
+    
+    expect(subscriptionCount).toBe(0);
+    
+    source.subscribe();
+    source.subscribe();
+    
+    expect(subscriptionCount).toBe(1);
+  });
+  
```

Comment:
Should we also have a marble test where `e1` is a cold Observable to make sense applying `share` on it? Also the case of multiple subscribers to a was-cold shared Observable makes a lot of sense to have a subscribe-and-unsubscribe marble diagram test that we mentioned in #428.